css/style.css

```diff
@@ -0,0 +1,64 @@
+html{
+
+font-size: 62.5%;    
+}
+
+h1{
+    text-align: center;
+    font-size: 4rem;
+    color: rgb(1, 78, 145);
+}
+.form-holder{
+   width: 60%;
+   margin: auto;
+}
+
+.description{
+    font-size: 3rem;
+    color: rgb(97, 100, 102);
+    text-align: center;
+
+}
+
+.input{
+    width: 80%;
+    padding: 2rem 2rem ;
+    font-size: 1.5rem;
+    border-radius: 15px;
+    outline: 0;
+    margin: 2rem 0;
+}
+
```

Comment:
Please remove the unnecessary spaces. Excessive spacing in a CSS documents causes Linters errors. I would suggest that you examine your entire document and adjust accordingly.